README.md

```diff
@@ -70,17 +70,11 @@ After you're finished please remove all the comments and instructions!
 ### Tech Stack <a name="tech-stack"></a>
 
 
-<details>
-  <summary>Client</summary>
-  <ul>
-    <li><a href="https://reactjs.org/">React.js</a></li>
-  </ul>
-</details>
 
 <details>
   <summary>Server</summary>
   <ul>
-    <li><a href="https://expressjs.com/">Express.js</a></li>
+    <li><a href="https://guides.rubyonrails.org/index.html">Ruby On Rails</a></li>
   </ul>
 </details>
 
```

Comment:
- [ ][optional]  @Cluab it would be nice if you added future features explaining what will be added in the app in future milestones, also your live demo link is wrong, kindly add correct one or leave a message 'coming soon'